drivers/linstorvhdutil.py

```diff
@@ -102,20 +102,25 @@ def wrapper(*args, **kwargs):
                     response = call_remote_method(
                         self._session, host_ref_attached, remote_method, device_path, remote_args
                     )
+                    util.SMlog(
+                        "Success accessing on attached node for device {} ({}): {}: {}".format(device_path, vdi_uuid, remote_method, str(response)),
```

Comment:
Maybe `Successful access`?
```suggestion
                        "Successful access on attached node for device {} ({}): `{}` = {}".format(device_path, vdi_uuid, remote_method, str(response)),
```

Reply:
We can use a function to log this message because it's duplicated in many parts.